scipy/stats/stats.py

```diff
@@ -1238,6 +1238,12 @@ def skewtest(a, axis=0, nan_policy='propagate'):
     -----
     The sample size must be at least 8.
 
+	References
+	----------
+	.. [1] D'Agostino, R. B., A. J. Belanger and R. B. D'Agostino, Jr 1990.
+	A suggestion for using powerful and informative tests of normality.
+	American Statistician 44: 316-321
```

Comment:
The indentation of this block is off, that's why the tests are failing. It should line up with the block above it.

Maybe you used tabs that were turned into 8 spaces instead of 4?